tasks/swig.rake

```diff
@@ -11,24 +13,32 @@
 def run_swig(mod)
   swig = `which swig`.chomp
   swig = `which swig2.0`.chomp if swig.empty?
+  swiglib = `swig -swiglib`.chomp
+  abort unless $?.success?
```

Comment:
Can we print a nice message here about why it failed? It fails if `swig` can't be found, right?